odxtools/diaglayer.py

```diff
@@ -454,49 +455,115 @@ def get_communication_parameter(self, name: str) \
 
         return cps[0]
 
+    def _get_comparam_value(self,
+                            com_param : CommunicationParameterRef) \
```

Comment:
I think it would be better to make this function part of the `CommunicationParameterRef` or Comparam/ComplexComparam
* It's not using "self"
* It's using only `com_param.value` and `com_param.comparam`

Reply:
thanks for the suggestions. I moved these methods to the `CommunicationParameterRef` class where they belong anyway...